src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/handlers/presubmit/ListCaseNotificationHandler.java

```diff
@@ -86,25 +87,28 @@ public boolean canHandle(PreSubmitCallbackStage callbackStage, Callback<AsylumCa
         final String caseId = asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class)
             .orElseThrow(() -> new IllegalStateException("Case ID for the appeal is not present"));
 
+        boolean adjournHearingWithoutDate = (callback.getEvent()
+            .equals(Event.ADJOURN_HEARING_WITHOUT_DATE));
+
         final HearingInstructMessage bundleInstructMessage =
             hearingInstructMessage()
                 .withConsumerReference(notificationsHelper.getConsumerReference(caseId))
                 .withHoReference(homeOfficeReferenceNumber)
                 .withMessageHeader(notificationsHelper.getMessageHeader())
                 .withMessageType(HEARING.name())
-                .withNote(getHearingNotificationContent(asylumCase))
-                .withHearing(getHearingData(asylumCase))
+                .withNote(getHearingNotificationContent(asylumCase, adjournHearingWithoutDate))
+                .withHearing(getHearingData(asylumCase, adjournHearingWithoutDate))
                 .build();
 
         log.info("Finished constructing {} notification request for caseId: {}, HomeOffice reference: {}",
             HEARING.toString(), caseId, homeOfficeReferenceNumber);
 
         final String notificationStatus = homeOfficeInstructService.sendNotification(bundleInstructMessage);
 
-        if (callback.getEvent().equals(Event.EDIT_CASE_LISTING)) {
-            asylumCase.write(AsylumCaseDefinition.HOME_OFFICE_EDIT_LISTING_INSTRUCT_STATUS, notificationStatus);
-        } else {
+        if (callback.getEvent().equals(Event.LIST_CASE)) {
             asylumCase.write(AsylumCaseDefinition.HOME_OFFICE_HEARING_INSTRUCT_STATUS, notificationStatus);
+        } else {
+            asylumCase.write(AsylumCaseDefinition.HOME_OFFICE_EDIT_LISTING_INSTRUCT_STATUS, notificationStatus);
```

Comment:
We need a separate status flag for adjourned hearing.

Reply:
Separate status flag created - "homeOfficeAdjournWithoutDateInstructStatus"